compiling/extract_and_compile.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+tar xzf NthPrime.tgz
+cd NthPrime
+gcc main.c nth_prime.c -o NthPrime
+./NthPrime $1
```

Comment:
`shellcheck` is grumpy here as well and you should fix the issue and/or ask for more info on why `shellcheck` is unhappy.